modules/dataquery/jsx/react.tabs.js

```diff
@@ -679,7 +682,7 @@ class StatsVisualizationTabPane extends Component {
         </table>
       );
 
-      let content = (
```

Comment:
This was causing the Stats tab to display content from the parent block (line 634)